opentelemetry-resourcedetector-gcp/README.rst

```diff
@@ -25,8 +25,30 @@ Installation
     pip install opentelemetry-resourcedetector-gcp
 
 ..
-    TODO: Add usage info here
 
+Usage
+------------
+
+.. code:: python
```

Comment:
Github's RST render thinks this is comment and hides everything. You might need an empty line or something